### PR TITLE
Skip flaky transport test

### PR DESCRIPTION
**Pull request by mikaelm12**
Issue: https://github.com/aspnet/AspNetCore/issues/8262 and https://github.com/aspnet/AspNetCore-Internal/issues/1910#event-2188177593

Looking into fixing the test but it's causing failures so need to skip now

CC @aspnet/brt

### CODE REVIEW

**Review by halter73 — APPROVED** [review]

**Comment by halter73** [comment]
The link to the failed build in the issue isn't working for me:

https://dev.azure.com/dnceng/public/_build/results?buildId=116342&view=logs

I see "The requested build 116342 could not be found." when I follow the link.

**Comment by mikaelm12** [comment]
> The link to the failed build in the issue isn't working for me:

That was a pr build. It was probably cleared out. Here are more recent issues https://github.com/aspnet/AspNetCore-Internal/issues/1886 https://github.com/aspnet/AspNetCore-Internal/issues/1910#event-2188177593

File: src/SignalR/clients/java/signalr/src/test/java/com/microsoft/signalr/HubConnectionTest.java [hunk 1]
```diff
@@ -1494,7 +1494,7 @@ public void afterSuccessfulNegotiateConnectsWithLongPollingTransport() {
         assertEquals("{\"protocol\":\"json\",\"version\":1}" + RECORD_SEPARATOR, sentMessages[0]);
     }
 
-    @Test
+    // Github issue:https://github.com/aspnet/AspNetCore/issues/8262
```
Comment (anurse) [inline src/SignalR/clients/java/signalr/src/test/java/com/microsoft/signalr/HubConnectionTest.java]: Can you use `@Ignore` ? http://junit.sourceforge.net/javadoc/org/junit/Ignore.html

Comment (anurse) [reply src/SignalR/clients/java/signalr/src/test/java/com/microsoft/signalr/HubConnectionTest.java]: Like we use `Skip` in xUnit.net:

```java
@Ignore("https://github.com/aspnet/AspNetCore/issues/8262")
@Test
...
```

**Comment by mikaelm12** [comment]
```
  Failed   E2ETests.PublishAndRunTests.PublishAndRun_Test(variant: Server: Kestrel, TFM: netcoreapp3.0, Type: Portable, Arch: x64)
  Error Message:
   System.Exception : dotnet publish exited with exit code : 1
  Stack Trace:
     at Microsoft.AspNetCore.Server.IntegrationTesting.ApplicationPublisher.Publish(DeploymentParameters deploymentParameters, ILogger logger) in /_/src/Hosting/Server.IntegrationTesting/src/ApplicationPublisher.cs:line 92
     at Microsoft.AspNetCore.Server.IntegrationTesting.ApplicationDeployer.DotnetPublish(String publishRoot) in /_/src/Hosting/Server.IntegrationTesting/src/Deployers/ApplicationDeployer.cs:line 87
     at Microsoft.AspNetCore.Server.IntegrationTesting.SelfHostDeployer.DeployAsync() in /_/src/Hosting/Server.IntegrationTesting/src/Deployers/SelfHostDeployer.cs:line 58
     at E2ETests.PublishAndRunTests.PublishAndRun_Test(TestVariant variant) in /home/vsts/work/1/s/src/MusicStore/test/MusicStore.E2ETests/PublishAndRunTests.cs:line 53
  --- End of stack trace from previous location where exception was thrown ---
```
cc @aspnet/brt A quick issue search with the test name didn't bring up any results.

**Comment by anurse** [comment]
aspnet/AspNetCore-Internal#1913

**Pull request merged**